masks2d.scad

```diff
@@ -37,6 +37,9 @@
 //   d = Diameter of the roundover.
 //   anchor = Translate so anchor point is at origin (0,0,0).  See [anchor](attachments.scad#subsection-anchor).  Default: `CENTER`
 //   spin = Rotate this many degrees around the Z axis after anchor.  See [spin](attachments.scad#subsection-spin).  Default: `0`
+// Side Effects:
+//   Sets the default tag to "remove".
```

Comment:
Suggest using more explicit description, like: "Tags the children with "remove" (and hence sets $tag) if no tag is already set."